### PR TITLE
fix(tui): keep fleet selection visible

### PR DESCRIPTION
**Pull request by igorsheg**
## Summary
- align Plot TUI render scheduling with pi-tui requestRender coalescing instead of a second 50ms throttle
- clamp the fleet work list to the terminal height so busy dashboards do not overflow
- keep the selected running work visible while moving through a large fleet and show overflow hints

## Verification
- bun run check

### CODE REVIEW

File: packages/tui/src/fleet-view.ts [hunk 1]
```diff
@@ -64,13 +64,40 @@ const activityGlyph = (tone: "ok" | "bad" | "info") =>
...
+		availableRows >= 3
+			? Math.max(0, selectedEnd - availableRows + 1)
+			: selectedStart;
+	const offset = Math.min(maxOffset, preferredOffset);
+	const visible = workLines.slice(offset, offset + availableRows);
+	if (offset === 0)
+		return [...visible.slice(0, -1), item(style.muted("    … more below"))];
+	if (offset >= maxOffset)
+		return [item(style.muted("    … more above")), ...visible.slice(1)];
+	return [
```
Comment (igorsheg) [inline packages/tui/src/fleet-view.ts:87]: This middle-overflow branch can evict the selected work label in the smallest supported viewports. For example, with `height: () => 14`, 8 running items, and item #5 selected, `availableRows` is 3; the slice contains the selected 3-line block, but this branch replaces the first and last lines with ellipses, leaving only `working · ...` and no `› ● #5 Item 5`. For tight windows, prefer keeping the selected label/marker and dropping one/both ellipsis indicators.

**Review by igorsheg — COMMENTED** [review]
## Plot Review

**Disposition:** COMMENT
**Verification:** Reviewed the TUI diff and surrounding render/height handling. Ran `bun test packages/tui/test/dashboard.test.ts` and `bun run check` successfully. Also reproduced the edge case with a 14-row dashboard render: selecting item #5 from 8 running items renders only ellipsis rows plus the stage line, not the selected label/marker.
**Head:** `0857063bfcb285222163b4ffff1eba2d8136e2db`

### Summary

Found one small-terminal viewport edge case to fix. The focused dashboard test and full `bun run check` pass, but the new clamp can still hide the selected row when only 1–3 work rows are available.

### Findings

- P1 `packages/tui/src/fleet-view.ts:86` — Selected row can be replaced by ellipsis rows in very small viewports. On terminals where the work viewport is only 1–3 rows tall, the selected work's label/marker can disappear, which defeats the purpose of keeping fleet selection visible. At 14 terminal rows with 8 running items and item #5 selected, the render contains `… more above`, `working · ...`, `… more below`, but not `› ● #5 Item 5`. Evidence: `clampWorkViewport` chooses an offset that includes the selected 3-line block, then the middle branch replaces the first and last visible lines with ellipsis indicators. With `availableRows === 3`, that removes the selected label and activity lines; with `availableRows <= 2`, it can render only ellipses. Reproduced via a local `PlotDashboard` render with `height: () => 14`. Fix: When space is this tight, prioritize the selected label/marker over both overflow indicators. For example, special-case `availableRows <= 3`, or reserve indicator rows before computing the slice so inserted ellipses cannot evict the selected label line.

### Confidence

High

**Review by igorsheg — COMMENTED** [review]
## Plot Review

**Disposition:** COMMENT
**Verification:** Reviewed the incremental diff from the previous Plot review and the full TUI viewport changes. Verified the prior 14-row selected-work regression is covered by the new tiny terminal assertion. Ran `bun test packages/tui/test/dashboard.test.ts` and `bun run check` successfully.
**Head:** `6dde3117a09d1dc53394c65d775f1b110c19a809`

### Summary

No remaining issues found. The new `availableRows <= 3` path preserves the selected fleet row instead of replacing it with overflow indicators, and the focused regression test exercises the previously reported edge case.

### Confidence

High
